test/unit/dat-worker-pool_tests.rb

```diff
@@ -60,44 +73,10 @@ class InitTests < UnitTests
       assert_equal [], subject.after_work_callbacks
     end
 
-    should "allow customizing it by passing options" do
-      num_workers = Factory.integer
-      logger      = Logger.new(STDOUT)
-      worker_pool = @worker_pool_class.new({
-        :num_workers  => num_workers,
-        :logger       => logger,
-        :do_work_proc => proc{ }
-      })
-      assert_equal num_workers, worker_pool.num_workers
-      assert_equal logger,      worker_pool.logger
-    end
-
-    should "demeter its queue callbacks" do
-      callback = proc{ }
-      subject.on_queue_pop(&callback)
-      assert_equal [callback], subject.on_queue_pop_callbacks
-      callback = proc{ }
-      subject.on_queue_push(&callback)
-      assert_equal [callback], subject.on_queue_push_callbacks
-    end
-
-    # TODO - stub queue once we can pass a queue instance to worker pool
-    should "know if its queue is empty or not" do
-      assert_true subject.queue_empty?
-      subject.queue.start
-      subject.queue.push Factory.string
-      assert_false subject.queue_empty?
-    end
-
-    # TODO - once we pass queue, test shutdown changes on queue
     should "start its queue when its started" do
-      assert_true subject.queue.shutdown?
-      subject.start
-      assert_false subject.queue.shutdown?
-      subject.shutdown
-      assert_true subject.queue.shutdown?
+      assert_true @queue.shutdown?
       subject.start
-      assert_false subject.queue.shutdown?
+      assert_false @queue.shutdown?
```

Comment:
@jcredding can we switch/invert to `running?` here?  Seems to flow better with "start" in test desc.  Also more "positive" - not running (sad), start it, now running (happy).